druid-shell/src/backend/windows/window.rs

```diff
@@ -1507,6 +1513,30 @@ impl WindowBuilder {
             Ok(handle)
         }
     }
+
+    /// When creating a sub-window, we need to scale its position with respect to its parent.
+    /// If there is any error while scaling, log it as a warn and show sub-window in top left corner of screen/window.
+    fn scale_sub_window_position(
+        un_scaled_sub_window_position: Option<Point>,
```

Comment:
You're only using this in one place, and you've already checked that there is a position, so the `Option` is redundant, no?

Reply:
Agree. Made this much simpler.